CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [v3.2.0] - 2019-11-10
+
+### Added:
+- **[dnsprovider]** Add support for autodns
+
+### Changed:
+- **[dnsprovider]** httpreq: Allow use environment vars from a _FILE file
```

Comment:
\_FILE breaks Markdown for me (at least in the diff view), because underscores are an alternative way to produce italics. Maybe enclose the whole token it with backticks (`_FILE`), or escape the underscore with a backslash (\_FILE)?